ui-src/src/components/Lobby.tsx

```diff
@@ -125,6 +127,7 @@ const GameList = ({ allGames }) => {
               <Link to={`/game/${hash}`}>
                 <td>{game.description}</td>
               </Link>
+              <td><svg width="30" height="30" data-jdenticon-value={game.creatorHash}/>{game.creatorHash.substring(0,11)}<svg width="30" height="30" data-jdenticon-value={game.creatorHash}/></td>
```

Comment:
You can just use `<Jdenticon size={30} hash={game.creatorHash} />` (see Cell.tsx for example)